src/pages/Login.tsx

```diff
@@ -1,18 +1,56 @@
-import React from "react";
-import ContainerElement from "../components/Container";
+import React, { ChangeEvent, FormEvent, useState } from 'react'
+import ContainerElement from '../components/Container'
+import {
+   ForgetPasswordLink,
+   FormContainer,
+   LoginContainer,
+} from '../components/Login/Login.style'
 
-export interface ILoginPageProps {
-}
+export interface ILoginPageProps {}
 
-const LoginPage: React.FunctionComponent<ILoginPageProps> = () => {
-    return (
-        <ContainerElement>
-            <label>Login</label>
-            <input type="email"/>
-            <input type="password"/>
-        </ContainerElement>
+export const LoginPage: React.FunctionComponent<ILoginPageProps> = () => {
+   const [email, setEmail] = useState<string>('')
+   const [password, setPassword] = useState<string>('')
 
-    )
-}
+   const handleClick = async (e: FormEvent) => {
+      e.preventDefault()
+      setEmail('')
+      setPassword('')
+   }
 
-export default LoginPage
\ No newline at end of file
+   return (
+      <ContainerElement>
+         <LoginContainer>
+            <img
+               src="https://static1.s123-cdn-static-a.com/uploads/5191798/400_609bb5e2d9a39.png"
+               alt="logo"
+            />
+            <FormContainer onSubmit={handleClick}>
```

Comment:
Ttry # handleSubmit is more descriptive.

Reply:
done 👍